next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
```

Comment:
There's a comment from Next.js here that says not to edit the file...

Are you sure that you want to make this change?

Reply:
Oh sorry, I didn't modify this myself. After removing the `@types/nprogress` one, that line got removed by next itself.

Reply:
That's fine then, if the program did it. 😉